lua/telescope/builtin/lsp.lua

```diff
@@ -198,8 +198,9 @@ lsp.diagnostics = function(opts)
     return
   end
 
+  opts.hide_filename = utils.get_default(opts.hide_filename, true)
   pickers.new(opts, {
-    prompt_title = 'LSP Diagnostics',
+    prompt_title = utils.get_default(opts.prompt_title, 'LSP Document Diagnostics'),
```

Comment:
This `utils.get_` shouldn't be necessary. We are always preferring the prompt title that is in opts.

Example: `Telescope find_files prompt_title=asdf

Reply:
...

fixed :D